marketing/copy.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Marketing Copy
+description: Text to use on sites which promote Mathesar
+published: true
+date: 2023-02-23T00:00:00.000Z
+tags: 
+editor: markdown
+dateCreated: 2023-02-23T00:00:00.000Z
+---
+
+## Tagline
+
+### Technical
+
+> Mathesar is an open source web application providing an intuitive spreadsheet-like interface to PostgreSQL databases.
+
+
+### Non-technical
+
+> Mathesar is an application for managing structured data from an intuitive spreadsheet-like web interface.
+
+
+## Description
+
+### Technical
+
+> `{tagline_technical}`
+>
+> Features such as foreign keys, data types, and schemas directly reflect the behavior of Postgres, allowing users of all skill levels to collaboratively work with the same normalized relational data that other applications might use too. You can model your data, alter its structure, quickly enter data, and even generate reports. Self-hosting Mathesar gives you control, ownership, and privacy of your data.
+
+### Non-technical
+
+_(varies depending on context)_
+
+
+## Mission
+
+> Our mission is to build a tool that lets people of all skill levels use data to solve complex problems.
+
+## Fundraising
+
+> `{mission}`
+>
+> As a non-profit project, Mathesar relies on donations to fund development. A small core team of paid engineers guide the direction of project and maintain its underlying architecture. Leveraging open source principles, the core team also amplifies its work by facilitating a broader community of volunteer contributors to help build the product.
```

Comment:
"relies on donations" seems a bit disingenuous to me because we're currently grant-funded and plan to be funded through operating a SaaS. I'd make this more along the lines of "Your donations will help us do more."

Reply:
> "relies on donations" seems a bit disingenuous

Good point. My intention here was to consider grants as a form of donation, but I agree that it's misleading as-written. I've pushed 331ee721e0313822680b55cf8155fc3671d75277 which I hope adds enough clarity.

> plan to be funded through operating a SaaS

The text after my changes won't be completely accurate if/once we launch a paid SaaS product, but I figure we can update the text as-needed.